pkg/util/removeall/removeall.go

```diff
@@ -106,3 +106,18 @@ func RemoveAllOneFilesystem(mounter mount.Interface, path string) error {
 	}
 	return err
 }
+
+// RemoveAllOneFilesystem removes the path and any children it contains, using
+// the os.Remove function. It makes sure it does not cross mount boundaries,
+// i.e. it does *not* remove files from another filesystem.
```

Comment:
clarify the behavior when files/folders under this path do cross filesystems... is an error returned? can some files still be removed before that error is returned?